public/panel/css/panel.css

```diff
@@ -272,3 +272,56 @@ table {
   width: 100%;
 }
 
+/*** Elrond Styling ***/
```

Comment:
Rather than put this in the panel CSS, would it be better to make a separate CSS file for Elrond?